### PR TITLE
OSDOCS-10433: Update ROSA Compliance page for HCP

### PR DESCRIPTION
**Pull request by bmcelvee**
Version(s):
4.15+ 

Issue:
https://issues.redhat.com/browse/OSDOCS-10433

Link to docs preview:
https://75454--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-policy-process-security.html#rosa-policy-compliance_rosa-policy-process-security

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bmcelvee** [comment]
@yuwang-RH PTAL, thank you!

File: modules/rosa-policy-security-regulation-compliance.adoc [hunk 1]
```diff
@@ -57,15 +57,17 @@ Any issues that may be discovered are prioritized based on severity. Any issues
...
 
-| PCI DSS | Yes | No
+| PCI DSS | Yes | Yes
 
-| SOC 2 Type 2 | Yes | No
+| SOC 1 Type 2 | Yes | No
+
+| SOC 2 Type 2 | Yes | Yes
 
 | SOC 3 | Yes | No
```
Comment (EricPonvelle) [inline modules/rosa-policy-security-regulation-compliance.adoc:72]: I confirmed that SOC3 should also be yes for ROSA with HCP

**Review by EricPonvelle — COMMENTED** [review]
Small change required

File: modules/rosa-policy-security-regulation-compliance.adoc (hunk 1 above)
Comment (bmcelvee) [reply modules/rosa-policy-security-regulation-compliance.adoc]: Updated, thank you!

**Comment by bmcelvee** [comment]
SME approval in slack.

**Pull request merged**